.github/workflows/build.yml

```diff
@@ -19,14 +19,11 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["7.4", "8.0"]
-                node: ["10.x"]
+                php: ["8.0"]
+                node: ["14.x"]
                 mysql: ["5.7", "8.0"]
                 symfony: ["^4.4", "^5.4"]
-                sylius: ["~1.10.0", "^1.11.0-alpha"]
-                exclude:
-                    - php: "7.4"
-                      sylius: "^1.11.0-alpha"
+                sylius: ["~1.10.11", "~1.11.2"]
```

Comment:
Maybe `1.10.*` and `1.11.*`? Would be the same but feels better without specific versions numbers